src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
+    }
+
+    /**
+     * @return string
```

Comment:
> I think it's easier to follow convention that we use docblocks everywhere rather than allowing for exceptions

@norzechowicz They still require an update when the code changes, which means extra work to maintain them.

They were quite useful before it was possible to typehint in code directly. They are still useful when putting some extra comment to clarify stuff that isn't obvious in the code. But now? 

Given they have no added value, what's the point?